src/pages/settings/Security/TwoFactorAuth/StepWrapper/StepWrapper.tsx

```diff
@@ -6,20 +6,18 @@ import HeaderWithBackButton from '@components/HeaderWithBackButton';
 import ScreenWrapper from '@components/ScreenWrapper';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as TwoFactorAuthActions from '@userActions/TwoFactorAuthActions';
-import StepWrapperPropTypes from './StepWrapperPropTypes';
+import type StepWrapperPropTypes from './StepWrapperPropTypes';
```

Comment:
move this type to that file and name it StepWrapperProps since this is our [guideline](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#naming-conventions)